qa/tasks/qemu.py

```diff
@@ -457,6 +459,12 @@ def run_qemu(ctx, config):
                         ),
                     ],
                 )
+        log.info("Deleting exported directory...")
+        for client in config.keys():
+            (remote,) = ctx.cluster.only(client).remotes.keys()
+            remote.run(args=[
+                'sudo', 'rm', '-r', '/export'
```

Comment:
Isn't this covered in `_teardown_nfs_mount` right now? Should that be removed from `_teardown_nfs_mount` to avoid the race as described?

Reply:
correct. this is to avoid the races between multiple RBD client in the same teutholgy-node. 
This can make ensure that the last qemu cleanup /export directory.

Reply:
The issue is that in the prior step, `_teardown_nfs_mount` is *already* deleting the full `/export' directory so in effect this does nothing.

Reply:
so sorry for missing deleting /export from teardown_nfs when rebase code. modified it. many thanks.